cmds/core/poweroff/poweroff_linux.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// poweroff turns the system off, without delay. There are no options.
+//
+// Synopsis:
+//     poweroff
+//
+// Description:
+//     poweroff calls the kernel to power off the systems.
+//
+package main
+
+import (
+	"log"
+
+	"golang.org/x/sys/unix"
+)
+
+func main() {
+	if err := unix.Reboot(unix.LINUX_REBOOT_CMD_POWER_OFF); err != nil {
```

Comment:
you might be able to make this non-linux if you used RB_POWER_OFF, which is the same on linux AFAIK.

though not sure if go has it, or if there's really a distinction between unix and linux in uroot.

Reply:
it's not in Go, so for now, since this is linux-specific anyway, I'll leave it with the linux constant and Daniel will surely fix it later :-)

This command is so damn handy